velox/expression/EvalCtx.cpp

```diff
@@ -172,6 +172,36 @@ auto throwError(const std::exception_ptr& exceptionPtr) {
 }
 } // namespace
 
+void EvalCtx::setStatus(vector_size_t index, Status status) {
+  VELOX_DCHECK(!status.ok());
```

Comment:
wonder if we should have an actual check. Otherwise in non-debug code an "ok" will be wrapped as a runtime error.